src/link-graph.spec.js

```diff
@@ -0,0 +1,9 @@
+import test from 'tape';
+import React from 'react';
+import LinkNodesExample from './example/link-graph-simple.example';
```

Comment:
Not sure if `*.example` is a right way to call examples here: the file is already in the example folder.